src/pilot/assign_role.py

```diff
@@ -817,35 +791,6 @@ def configure_raid(ironic_client, node_uuid, super_role, os_volume_size_gb,
     # Set the target RAID configuration on the ironic node.
     ironic_client.node.set_target_raid_config(node_uuid, target_raid_config)
 
-    # Workaround 4:
-    # Attempt to convert all of the physical disks in the target RAID
-    # configuration to RAID mode. This may succeed or fail. A
-    # controller's capability to do that, or lack thereof, has no
-    # bearing on success or failure.
-    controllers_to_physical_disk_ids = defaultdict(list)
-
-    for logical_disk in target_raid_config['logical_disks']:
-        # Not applicable to JBOD logical disks.
-        if logical_disk['raid_level'] == 'JBOD':
-            continue
-
-        for physical_disk_name in logical_disk['physical_disks']:
-            controllers_to_physical_disk_ids[
-                logical_disk['controller']].append(physical_disk_name)
-
-    LOG.info("Converting physical disks configured to back RAID logical disks "
-             "to RAID mode")
-    succeeded = change_physical_disk_state_wait(
-        node_uuid, ironic_client, drac_client, 'RAID',
-        controllers_to_physical_disk_ids)
-
-    if succeeded:
-        LOG.info("Completed converting physical disks configured to back RAID "
-                 "logical disks to RAID mode")
-    else:
-        LOG.critical("Attempt to convert physical disks configured to back "
-                     "RAID logical disks to RAID mode failed")
-        return False
 
```

Comment:
Please delete this blank line, too. My Python-aware Eclipse PyDev editor complains loudly about extra lines.